Src/IronPython/Runtime/Operations/StringOps.cs

```diff
@@ -133,8 +133,8 @@ public class ExtensibleString : Extensible<string>, ICodeFormattable, IStructura
         }
 
         public virtual bool __contains__(object value) {
-            if (value is string) return Value.Contains((string)value);
-            else if (value is ExtensibleString) return Value.Contains(((ExtensibleString)value).Value);
+            if (value is ExtensibleString es) return Value.Contains(es.Value);
+            if (value is string s) return Value.Contains((string)value);
```

Comment:
nit: `Value.Contains(s)`

Reply:
good catch - that was the whole point of the change to avoid double casting